dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateFactoryImpl.java

```diff
@@ -317,18 +318,18 @@ else if(resultList.size() < internalLimit)
 
 	}
 	@Override
-	public List<HTMLPage> getPagesUsingTemplate(Template template) throws DotDataException {
-		HibernateUtil hu = new HibernateUtil(HTMLPage.class);
+	public List<HTMLPageAsset> getPagesUsingTemplate(Template template) throws DotDataException {
+		HibernateUtil hu = new HibernateUtil(HTMLPageAsset.class);
```

Comment:
this won't work , HTMLPageAssets is not in the hibernate mapping. Let's get rid of the entire getPagesUsingTemplate method from here and all the Factory/API levels